### PR TITLE
Update 03.methods.md SNV/SV/classifier/panel

### PR DESCRIPTION
**Pull request by jharenza**
#### Purpose

_Briefly describe your changes here._
-Modify CNV methods
-Remove LUMPY methods
-Add stub for germline calls (since used in BAF calculations)
-Updated Medulloblastoma classifier link
-Clarify panel methods

#### Issue

_What GitHub issue does your pull request address?_
#47 , #49 

#### Pull review checklist

The PR will be considered ready for review when all four items have been checked.

- [X] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [X] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [X] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [X] All figures follow the [Manubot figure instructions](https://github.com/jaclyn-taroni/OpenPBTA-manuscript/blob/master/USAGE.md#figures)

### CODE REVIEW

**Comment by jharenza** [comment]
Will make the clinical PR separate.

File: content/03.methods.md [hunk 1]
```diff
@@ -99,15 +103,18 @@ We filtered outputs from both callers on the "PASS" filter, and annotated using
 
 ### Somatic Copy Number Variant Calling
 
-We used Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps] v8.7 and CNVkit [@doi:10.1371/journal.pcbi.1004873] v0.9.3 for copy number variant calls. 
-CNVkit was run using default parameters on human genome reference hg38 and using the batch command for tumor-normal pairs rather than a panel of normals.
+We used Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps] v11.6 and CNVkit [@doi:10.1371/journal.pcbi.1004873] v0.9.3 for copy number variant calls. 
+For both algorithms, the `germline_sex_estimate` (described below) was used as input for sample sex and germline variant calls were used as input for B-allele frequency (BAF) estimation. 
+ControlFreeC was run on human genome reference hg38 using the optional parameters of a 0.05 coefficient of variation, ploidy choice of 2-4, and BAF adjustment for tumor-normal pairs.
+Theta2 [doi:10.1093/bioinformatics/btu651] used VarDict germline and somatic calls, filtered on PASS and strongly somatic, to infer tumor purity.
```
Comment (jaclyn-taroni) [inline content/03.methods.md]:
```suggestion
Theta2 [@doi:10.1093/bioinformatics/btu651] used VarDict germline and somatic calls, filtered on PASS and strongly somatic, to infer tumor purity.
```

Comment (jaclyn-taroni) [reply content/03.methods.md]: Why the VarDict calls? It appears to be more permissive than the other callers given the way it was run for this project. Any intuition as to how using another caller would change the results (or perhaps Theta2 requires VarDict)?

File: content/03.methods.md [hunk 2]
```diff
@@ -75,11 +75,15 @@ For RNA-Seq, read strandedness was determined by running the [`infer_experiment.
 If calculated strandedness did not match strandedness information received from the sequencing center, samples were removed from analysis.
 We required at least 60% of RNA-Seq reads mapped to the human reference or samples were removed from analysis.
 
-### Somatic Single Nucleotide Variant Calling
+### Germline Variant Calling
+#### SNV and INDEL calling
+We used GATK Haplotype caller...
```
Comment (jaclyn-taroni) [inline content/03.methods.md]:
```suggestion
We used GATK Haplotype caller.
```
Could probably just leave it like that for the moment.

**Comment by jharenza** [comment]
@jaclyn-taroni Theta2 requires vardict - we tried with another algorithm and it did not work, and we didn't dig into it much.

**Pull request merged**